src/cli/cli.py

```diff
@@ -103,6 +111,22 @@ def help(self, cmd: List[str]) -> None:  # pylint: disable=unused-argument
 """
         self.out_stream.write(help_str)
 
+    def sftp(self, cmd: List[str]) -> None:
+        name = cmd[0]
+        comp = re.compile(CONTAINER_NAME_REGEX)
+        if not comp.match(name):
+            self.out_stream.write(f"'{name}' is not a valid container name\n")
+            return
+        self.container_manager.sftp(name)
+
+    def view_files(self, cmd: List[str]) -> None:
```

Comment:
If I run this on my install without Filezilla installed, it gives unhelpful exception:

```FileNotFoundError: [Errno 2] No such file or directory: PosixPath('/home/fcorp/Documents/Jabberwocky/container_manager/contrib/filezilla/bin/filezilla')```